update-readme.sh

```diff
@@ -0,0 +1,19 @@
+#!bash
+
+# A small script that updates the README for releases.
+# Arguments to the script
+# 1. the new release-hash
+# 2. the version number
+
+hash=$1
+version=$2
+
+OLD_HASH="OLD-HASH"
```

Comment:
It is a little bit annoying that if something gets out of sync here, this just stops working. It probably would be cleaner if you just had markers (not the old values) in some kind of template that you could replace. 

It would be perfectly okay to have a checked-in filled-in version of the README.

Reply:
done.